tests/testthat/test-palaeorotate.R

```diff
@@ -44,6 +44,29 @@ test_that("palaeorotate() point method works", {
                                  method = "point",
                                  model = "PALEOMAP")), nrow(occdf))
 
+  # Check handling of temporal range
+  occdf <- data.frame(lng = runif(10, -180, 180),
+                      lat = runif(10, -90, 90),
+                      age = rep(500, 10))
+  expect_true(
+    all(
+      is.na(
+        palaeorotate(occdf = occdf,
+                     method = "point",
+                     model = "SETON2012")$p_lng
+        ))
+  )
+
+  expect_true(
+    all(
+      is.na(
+        palaeorotate(occdf = occdf,
+                     method = "point",
+                     model = c("SETON2012", "MULLER2016"),
+                     uncertainty = TRUE)$max_dist
+      )
```

Comment:
Hmm...is this the desired behavior? Or should we use `na.rm = T` in our calculations?

Reply:
None of these models extend back to the opted age of rotation which is why NAs are returned here... in the function we only return NAs if one or no models are available for comparison

Reply:
Ah, I had only tried two models (one of which had an invalid age)! Upon further investigation this seems to work well for 2+ models with valid gplates numbers.